.gitpod.Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM gitpod/workspace-full
```

Comment:
We should use a lighter workspace eventually to speed up the prebuilt process. The current `workspace/full` supports a ton of tools we don't need.

Reply:
Can one of you file an issue on this for follow-up?  We'll never remember.

Reply:
Filed: #2564